miden-lib/asm/miden/kernels/tx/faucet.masm

```diff
@@ -97,7 +101,8 @@ proc.burn_fungible_asset
     # => [amount, TOTAL_ISSUANCE, ASSET]
 
     # compute new total issuance
-    sub exec.account::get_faucet_storage_data_slot exec.account::set_item dropw
+    # note: storage slot is known here, so set_item_raw can be used
+    sub exec.account::get_faucet_storage_data_slot exec.account::set_item_raw dropw
```

Comment:
Similar comment to the one above and this is also relevant to the other changes in this file.